ISISComputingGroup/EPICS-ioc
# Ticket 2998: Removed flow control on linmot

### Description of work

Removed the flow control settings.

### To test

See https://github.com/ISISComputingGroup/IBEX/issues/2998

Confirm that the linmot connected to NDXDEMO ( which has been running for a while with these settings) still works without taking a long time to write.

See also https://github.com/ISISComputingGroup/EPICS-motor/pull/24

---

#### Code Review

    - [PV naming](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/PV-Naming).
    - [Disable records](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Disable-records)
    - [Record simulation](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Record-Simulation)
    - [Finishing touches](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/IOC-Finishing-Touches)

### Functional Tests

- IOC responds correctly in:

### Final steps
